src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -135,7 +135,7 @@ def get_issue_table_contents(issue_list: List[Dict[str, int]]) -> List[PullReque
 # TODO(cathy): Change the client typing to allow for multiple SCM Integrations
 def safe_for_comment(
     gh_client: GitHubAppsClient, repository: Repository, pull_request: PullRequest
-) -> Tuple[bool, List[Dict[str, str]]]:
+) -> List[Dict[str, str]]:
```

Comment:
@ykamo001 applied the suggestions from your comment here! https://github.com/getsentry/sentry/pull/61981#discussion_r1432020223